js/components/request_summary_contact_section.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function RequestSummaryContactSection({ formData }) {
+  const data = formData.requester_contact;
+  return (
+    <div className="request-summary_section">
+      <h5>Name</h5>
+      <p>{[data.first_name, data.last_name].join(' ')}</p>
```

Comment:
These fields should be `name_first`, `name_last`.

Reply:
`name_first` and `name_last` were returning blank for me, while these returned data. I'm not clear on what needs to change to make this work.

Reply:
yeah, this has been a long awaited change. First, we were waiting on https://github.com/usdoj/foia-api/pull/97 to be merged, which required some back and forth and review with OIP and design. Even now that it's merged, we still need to create the new forms on the back stage, something we could only have done _after_ 97 was merged.

OIP said they would be adding a few forms from the survey results on Monday/Tuesday and this will have the proper fields. Of course, an agency could come in and wreck their form if they chose to do so.